storage-proto/proto/confirmed_block.proto

```diff
@@ -59,6 +59,7 @@ message TransactionStatusMeta {
     repeated bytes loaded_readonly_addresses = 13;
     ReturnData return_data = 14;
     bool return_data_none = 15;
+    optional uint64 compute_units_consumed = 16;
```

Comment:
Protobuf 3.15 allows the `optional` field. Looks like the CI runner protoc is too old